### PR TITLE
Release 3.3.0 to dev

### PR DESCRIPTION
**Pull request by tofu-rocketry**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,16 @@
 Changelog for ssm
 =================
+* Thu Jun 29 2023 Adrian Coveney <adrian.coveney@stfc.ac.uk> - 3.3.0-1
+ - Added destination queue to the log during startup to aid troubleshooting.
+ - Added check that the config file exists to allow for better error messages.
+ - Changed dependencies to limit python-ldap below 3.4.0 and python-daemon below 2.3.0.
+ - Changed rpmbuild config to use less OS-specific dependencies.
+ - Fixed read timeouts leading to a crash.
+ - Fixed command line to allow a relative file path for the DNs file.
```
Comment (rowan04) [inline CHANGELOG:9]: could this line be worded better? the command line itself hasn't been fixed?

Comment (tofu-rocketry) [reply CHANGELOG:9]: How about, "Fixed command line arguments to..." ?

Comment (rowan04) [reply CHANGELOG:9]: Yeah 👍

Comment (tofu-rocketry) [reply CHANGELOG:9]:
```suggestion
 - Fixed command line arguments to allow a relative file path for the DNs file.
```

File: apel-ssm.spec [hunk 2]
```diff
@@ -100,6 +100,17 @@ rm -rf $RPM_BUILD_ROOT
 %doc %_defaultdocdir/%{name}
 
 %changelog
+* Thu Jun 29 2023 Adrian Coveney <adrian.coveney@stfc.ac.uk> - 3.3.0-1
+ - Added destination queue to the log during startup to aid troubleshooting.
+ - Added check that the config file exists to allow for better error messages.
+ - Changed dependencies to limit python-ldap below 3.4.0 and python-daemon below 2.3.0.
+ - Changed rpmbuild config to use less OS-specific dependencies.
+ - Fixed read timeouts leading to a crash.
+ - Fixed command line to allow a relative file path for the DNs file.
```
Comment (tofu-rocketry) [inline apel-ssm.spec:109]:
```suggestion
 - Fixed command line arguments to allow a relative file path for the DNs file.
```

**Review by rowan04 — APPROVED** [review]
lgtm

**Pull request merged**